spec/acceptance/force_spec.rb

```diff
@@ -91,6 +91,33 @@
     end
   end
 
+  describe 'when run should force merge nested arrays' do
+    let(:pp) do
+      <<-MANIFEST
+        concat { '#{@basedir}/file':
+          format => 'json',
+          force => true,
+        }
+
+        concat::fragment { '1':
+          target  => '#{@basedir}/file',
+          content => '{"one": [1]}',
+        }
+
+        concat::fragment { '2':
+          target  => '#{@basedir}/file',
+          content => '{"one": [2]}',
+        }
+      MANIFEST
+    end
+
+    it 'applies the manifest twice with no stderr' do
+      idempotent_apply(pp)
+      expect(file("#{@basedir}/file")).to be_file
+      expect(file("#{@basedir}/file").content).to match '{"one":[1,2]}' || '{"one":[1,2]}\n'
```

Comment:
could it be ```expect(file("#{@basedir}/file").content).to match '{"one":[1,2]}\R?'``` ?

Reply:
https://github.com/puppetlabs/puppetlabs-concat/blob/master/spec/acceptance/newline_spec.rb#L51 ?

Reply:
what about `.to contain '{"one":[1,2]}'` we dont specifically care if the new line is there or not,

Reply:
👍